doc/manual/index.adoc

```diff
@@ -62,35 +64,112 @@ Here's some sample code corresponding to the above diagram:
+=== Calling `(lacinia/execute)`
 
-=== `(stillsuit/execute)`
+Invoking lacinia to execute a GraphQL query is as simple as passing the values returned by
+`(stillsuit/decorate)` to `(lacinia/execute)`, along with the query you want to execute and its
+associated variables, if any.
+
+[source,clojure]
+----
+(let [options   #:stillsuit{:connection my-connection
+                            :resolvers  my-resolvers
+                            :schema     {:objects {:foo {...}}}}
+      decorated (stillsuit/decorate options)]
+  (lacinia/execute
+   (:stillsuit/schema decorated)                            ; Schema
+   "query getFoo(id: Int!) { foo(id: $id) { bar } }"        ; Query
+   {:foo-id 123}                                            ; Variables
+   (:stillsuit/app-context decorated)))                     ; Context
+----
+
+NOTE: By default, stillsuit will run
+https://lacinia.readthedocs.io/en/latest/overview.html#compiling-the-schema[`(lacinia.schema/compile)`]
+for you. If you need to mess with the schema before you compile it, you can set
+the value `:stillsuit/compile?` to `false` in the `:stillsuit/config` map.
+
+==== `(stillsuit/execute)`
 
 As a convenience, stillsuit also includes a
 http://docs.workframe.com/stillsuit/current/doc/stillsuit.core.html#var-execute[little wrapper function]
-which calls `(lainia/execute)` for you from the result of `(stillsuit/decorate)`.
+which calls `(lacinia/execute)` for you from the result of `(stillsuit/decorate)`.
+
+With `(stillsuit/execute)`, the above code would look like this:
+
+[source,clojure]
+----
+(let [options   #:stillsuit{:connection my-connection
+                            :resolvers  my-resolvers
+                            :schema     {:objects {:foo {...}}}}
+      decorated (stillsuit/decorate options)]
+  (stillsuit/execute
+   decorated                                                ; Stillsuit result
+   "query getFoo(id: Int!) { foo(id: $id) { bar } }"        ; Query
+   {:foo-id 123}))                                          ; Variables
+----
 
 == Configuring stillsuit
 
-(TBD)
+Stillsuit can take in a number of options to configure how it operates, which are
+represented as a map.
+
+The configuration settings come from three places:
+
+* Stillsuit has a set of defaults for most config settings.
+* If the schema configuration passed to stillsuit contains a top-level key `:stillsuit/config`,
+  those values will override the default values.
+* A map passed as the `:stillsuit/config` key in the options map of `(stillsuit/decorate)` will
+  override both of the above values.
+
+The config settings from all three places are deep-merged.
+
+=== Config settings
+
+`:stillsuit/compile?` - if set to `false`, stillsuit won't compile the lacinia schema configuration
+returned in the `:stillsuit/schema` key of `(stillsuit/decorate)`.
 
-* config file
-** sample config or link to one
-** loading the config
+(More TBD)
 
 == The `:stillsuit/ref` resolver
```

Comment:
Not sure if you should say something about how things just work, most of the time, for primitive types. You allude to it below, but I think it needs a little more context.

Reply:
Yeah, I'm not done with that yet but I wanted to put it in the "default resolver" section.